DOC-56 Import Technology Providers Playbook (#110)
Ed-Fi-Alliance-OSS/ed-fi-alliance-oss.github.io: docs/getting-started/provider-playbook/technology-providers-implementation/technology-partner-implementation/readme.md

# Technology Partner - Implementation

> This section contains best practice guidance on implementing an API client to support regular and secure data exchange. This best practice has been observed from current Ed-Fi field work. If you have other suggestions for best practices, please send us your thoughts by commenting on this page or submitting a ticket to [Ed-Fi Tracker](https://tracker.ed-fi.org/) and we will continue to add to this section!

Reviewer: ```suggestion
This section contains best practice guidance on implementing an API client to support regular and secure data exchange. This best practice has been observed from current Ed-Fi field work. If you have other suggestions for best practices, please send us your thoughts by submitting a case in the [Ed-Fi Community Hub](https://community.ed-fi.org) and we will continue to add to this section!
```

Author: updated.
